adapter/repository/in-memory-simple/src/main/java/org/mobilitydata/gtfsvalidator/execution/InMemoryExecutionParameterRepository.java

```diff
@@ -0,0 +1,52 @@
+package org.mobilitydata.gtfsvalidator.execution;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.commons.cli.DefaultParser;
+import org.apache.commons.cli.Options;
+import org.mobilitydata.gtfsvalidator.domain.entity.ExecutionParameter;
+import org.mobilitydata.gtfsvalidator.parser.ApacheExecutionParameterParser;
+import org.mobilitydata.gtfsvalidator.parser.JsonExecutionParameterParser;
+import org.mobilitydata.gtfsvalidator.usecase.port.ExecutionParameterRepository;
+
+import java.util.Collections;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+public class InMemoryExecutionParameterRepository implements ExecutionParameterRepository {
+    private final Map<String, ExecutionParameter> executionParameterCollection = new LinkedHashMap<>();
+    private final String[] arguments;
+
+    public InMemoryExecutionParameterRepository(String[] arguments) {
+        this.arguments = arguments;
+    }
+
+    @Override
+    public ExecutionParameter getExecutionParameterByShortName(final String executionParameterShortName) {
+        return executionParameterCollection.get(executionParameterShortName);
+    }
+
+    @Override
+    public Map<String, ExecutionParameter> getExecutionParameterCollection() {
+        return Collections.unmodifiableMap(executionParameterCollection);
+    }
+
+    @Override
+    public ExecutionParameter addExecutionParameter(final ExecutionParameter newExecutionParameter) {
+        executionParameterCollection.put(newExecutionParameter.getShortName(), newExecutionParameter);
+        return newExecutionParameter;
+    }
+
+    @Override
+    public boolean hasExecutionParameter(String shortName) {
```

Comment:
I could see users using shortName for command-line parameters, but longName for config files (to make them more understandable). The best solution would probably be to have a key property for each `ExecutionParameter` (which could be a string), and then if the long or short name is provided by the user store the `ExecutionParameter` keyed on this (instead of being keyed on the shortName or longName. Then, you can always just query by the key instead of having to support querying for both long and short name.